packages/useTrie/src/index.ts

```diff
@@ -22,8 +22,10 @@ interface ITrie {
 type TrieActionType = 'ADD' | 'REMOVE';
 interface TrieAction {
   type: TrieActionType;
-  word: Word;
-  trie: ITrie;
+  payload: {
```

Comment:
Refactor to [flux standard actions](https://github.com/redux-utilities/flux-standard-action#actions).

Reply:
I've actually never used Redux per se 
& just bought Dave Ceddia's Pure Redux course now (I've learned React from his Pure React & loved it)

I hope to pick up the standard while learning the course.